src/events/Raw.ts

```diff
@@ -32,7 +33,7 @@ export default class extends Listener<"raw"> {
 
         this.base.log.debug({ parsed, dataType });
 
-        if (parsed?.game_version && parsed?.game_version !== this.base.cdn.version)
+        if (parsed?.game_version && parsed?.game_version !== this.base.cdn.version && !Config.bypassVersionCheck)
```

Comment:
for the Config better use `base.config.bypassVersionCheck` instead of importing config back